cognite/client/_api/sequences.py

```diff
@@ -314,8 +319,20 @@ def insert(
                 >>> c = CogniteClient()
                 >>> data = {123 : ['str',3], 456 : ['bar',42] }
                 >>> c.sequences.data.insert(column_external_ids=['stringColumn','intColumn'], rows=data, id=1)
+
+            Finally, they can be a SequenceData object retrieved from another request. In this case column_external_ids from this object are used as well.
+
+                >>> from cognite.client.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> data = c.sequences.data.retrieve(id=2,start=0,end=10)
+                >>> c.sequences.data.insert(rows=data, id=1)
         """
         utils._auxiliary.assert_exactly_one_of_id_or_external_id(id, external_id)
+        if isinstance(rows, SequenceData):
```

Comment:
Perhaps something to implement for datapoints.insert() aswell?

Reply:
yes, I was thinking about it, but it's a bit trickier with DatapointsList and so on. I'll add just Datapoints